logstash-core/src/main/java/org/logstash/instrument/witness/MetricSerializer.java

```diff
@@ -36,7 +36,7 @@
          */
         static MetricSerializer<Metric<Long>> longSerializer(JsonGenerator gen) {
             return m -> {
-                if (m != null && m.isDirty() && m.getValue() != null) {
+                if (m != null && m.getValue() != null) {
```

Comment:
I just noticed that by checking the null value here, we have are still conditionally showing some of the values. Any gauge value, not yet set or not initialized will still conditionally appear since the initial state is null. Counters will always show. 

Update in-coming.